src/app/plan/duck/operations.ts

```diff
@@ -127,8 +128,15 @@ const addPlan = migPlan => {
       dispatch(addPlanSuccess(createRes.data));
 
       console.debug('Beginning PV polling');
-
+      let timesRun = 0;
       const interval = setInterval(async () => {
+        timesRun += 1;
+        if (timesRun === 3) {
+          clearInterval(interval);
+          dispatch(commonOperations.alertErrorTimeout('No PVs found'));
+          dispatch(pvFetchFailure());
+        }
```

Comment:
this is where we would trigger a validation check for this wizard step & update the wizard next enable value.